src/base.c

```diff
@@ -573,7 +594,7 @@ b0_alloc_tcache_stack(tsdn_t *tsdn, size_t stack_size) {
 	b0_alloc_header_size(&header_size, &alignment);
 
 	size_t alloc_size = sz_s2u(stack_size + header_size);
-	void *addr = base_alloc_impl(tsdn, base, alloc_size, alignment, &esn);
+	void *addr = base_alloc_impl(tsdn, base, alloc_size, alignment, &esn, NULL);
```

Comment:
nit: exceeded the 80 char max len (the tab length is 8 which might be different from your editor's default). Also a few places in ctl.c. 

Eventually we plan to switch to clang format so these can be automated.